brooklyn-server/core/src/main/java/org/apache/brooklyn/util/core/task/DynamicSequentialTask.java

```diff
@@ -309,7 +326,7 @@ public T call() throws Exception {
                                         }
 
                                         if (!primaryFinished && failureHandlingConfig.cancelPrimaryOnSecondaryFailure) {
-                                            cancel(true, false, false);
+                                            cancel(TaskCancellationMode.INTERRUPT_TASK_BUT_NOT_SUBMITTED_TASKS, false);
```

Comment:
Wow, this code is winning a prize for one of the biggest indents, which is a bit of a code smell!

But nothing to do with this PR, so ignore here.